docs/user-guide/environment-variables.md

```diff
@@ -71,3 +76,21 @@ For example, given a REST API that provides access to different types of record,
 a subflow could be created to access the API and handle the response, using an
 environment variable to identify which type of record should be accessed. Individual
 instances of the Subflow can then be customised for those particular types.
+
+### Flow/Group level environment variables
+
+Since [version 2.1](https://nodered.org/blog/2021/10/21/version-2-1-released#flowgroup-level-environment-variables), environment variables can be set at the flow or group level. This is done in the appropriate tab in the edit dialog for the flow or group.
+
+### Raspberry Pi OS considerations
+Users who have installed Node-RED on a Raspberry Pi using the provided [script](https://nodered.org/docs/getting-started/raspberrypi) and who run it as a service on booting or by using the `node-red-start` command should be aware that their flows will not have access to environment variables that are defined only in the calling process. In this case, environment variables can be defined in the settings file by statements of the form 
```

Comment:
```suggestion
When Node-RED is running as a service having been installed using the provided [script](https://nodered.org/docs/getting-started/raspberrypi), it will not have access to environment variables that are defined only in the calling process. In this instance, environment variables can be defined in the settings file by adding:
```